doc/020_installation.rst

```diff
@@ -188,8 +188,20 @@ are considered stable and releases are made regularly in a controlled manner.
 There's both pre-compiled binaries for different platforms as well as the source
 code available for download. Just download and run the one matching your system.
 
-The official binaries can be updated in place using the ``restic self-update``
-command (needs restic 0.9.3 or later):
+On your first installation, if you desire, you can verify the integrity of your
+downloads by testing the SHA-256 checksums listed in ``SHA256SUMS`` and verifying
+the integrity of the file ``SHA256SUMS`` with the PGP signature in ``SHA256SUMS.asc``. 
+The PGP signature was created using the key (`0x91A6868BD3F7A907 <https://pgp.mit.edu/pks/lookup?op=get&search=0xCF8F18F2844575973F79D4E191A6868BD3F7A907>`__):
```

Comment:
As the key server seems to be broken, let's just use the key stored at `https://restic.net/gpg-key-alex.asc`

Reply:
I changed the URL to the GPG keyfile. Thank you for the hint.